modules/javafx.controls/src/main/java/javafx/scene/control/ControlUtils.java

```diff
@@ -25,20 +25,18 @@
 
 package javafx.scene.control;
 
+import com.sun.javafx.scene.control.ReadOnlyUnbackedObservableList;
 import com.sun.javafx.scene.control.skin.Utils;
 import javafx.collections.ListChangeListener;
 import javafx.collections.ObservableList;
 import javafx.event.Event;
 import javafx.scene.Node;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
-
-import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 class ControlUtils {
-    private ControlUtils() { }
```

Comment:
The private constructor needs to be restored, since it is the pattern we use for static helper classes that should not be instantiated.